models/repo_list.go

```diff
@@ -241,10 +240,8 @@ func SearchRepositoryByName(opts *SearchRepoOptions) (RepositoryList, int64, err
 		return nil, 0, fmt.Errorf("Repo: %v", err)
 	}
 
-	if !opts.IsProfile {
-		if err = repos.loadAttributes(sess); err != nil {
-			return nil, 0, fmt.Errorf("LoadAttributes: %v", err)
-		}
+	if err = repos.loadAttributes(sess); err != nil {
```

Comment:
Wouldn't this make it slower by always loading the attributes?

Reply:
because of `IsProfile` only use in user profile so we can remove this condition.